src/quo2/components/switchers/card_main_info/style.cljs

```diff
@@ -0,0 +1,8 @@
+(ns quo2.components.switchers.card-main-info.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def title
+  {:color colors/white})
```

Comment:
do we need to set `color` here? these components should only be working in dark theme so we don't need to set items so it works in `light` theme.